src/libraries/Common/src/System/TimeProvider.cs

```diff
@@ -136,7 +136,7 @@ public TimeSpan GetElapsedTime(long startingTimestamp, long endingTimestamp)
         /// The newly created <see cref="ITimer"/> instance.
         /// </returns>
         /// <exception cref="ArgumentNullException"><paramref name="callback"/> is null.</exception>
-        /// <exception cref="ArgumentOutOfRangeException">The number of milliseconds in the value of <paramref name="dueTime"/> or <paramref name="period"/> is negative and not equal to <see cref="Timeout.Infinite"/>, or is greater than <see cref="int.MaxValue"/>.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">The number of milliseconds in the value of <paramref name="dueTime"/> or <paramref name="period"/> is negative and not equal to <see cref="Timeout.Infinite"/>, or is greater than <see cref="int.MaxValue">int.MaxValue</see>.</exception>
```

Comment:
Is this working without the `T:` prefix?

Reply:
This should probably be either `T:int.MaxValue` or `T:System.Int32.MaxValue`, no?

Reply:
And the added text `>int.MaxValue</see>` can be removed after that.

Reply:
Oh I'm sorry, this is the runtime repo. Disregard my comment about adding a `T:` prefix.